docs-examples/v1.6/quickstart/README.md

```diff
@@ -0,0 +1,48 @@
+# MongoDB Kafka Connector Lite
+
+This docker compose starts the following services:
+
+- MongoDB single node replica set
+- Kafka Connect
+- Kafka Broker
+- Kafka Zookeeper
```

Comment:
question: What do these services do, exactly? How do they get a document from source to sink, for example?

Reply:
Added the copy from the quickstart which describes the architecture of the sample pipeline